### PR TITLE
1048 feature/allow a user to see when a tile is new

### PR DESCRIPTION
**Pull request by BarryGeeMOJ**
### Context

> Does this issue have a Trello card?
https://trello.com/c/mTNr3CTH/1048-allow-a-user-to-see-when-a-tile-is-new

> If this is an issue, do we have steps to reproduce?
n/a

### Intent

> What changes are introduced by this PR that correspond to the above card?
- Add the 'new' tag to tiles with content that is 2 days old or newer
- Update and add new supporting tests

> Would this PR benefit from screenshots?
Screenshot 2022-06-21 at 13 03 02

### Considerations

> Is there any additional information that would help when reviewing this PR?
n/a

> Are there any steps required when merging/deploying this PR?
n/a

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: assets/sass/components/_content-tile-small.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.govuk-hub-contentTileSmall {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  align-items: center;
+  margin: 30;
```
Comment (jonhindle) [inline assets/sass/components/_content-tile-small.scss:6]: can we use a gov uk value for consistency?

File: assets/sass/components/_content-tile-small.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.govuk-hub-contentTileSmall {
```
Comment (jonhindle) [inline assets/sass/components/_content-tile-small.scss:1]: The rest of the tile CSS is in `_home-content.scss` rightly or wrongly. (A victim of evolution I think)
Should we try and keep this CSS together?

Comment (BarryGeeMOJ) [reply assets/sass/components/_content-tile-small.scss:1]: i agree the tile css should be kept together. Whether that should be in the home-content scss is another matter. As the tiles are used in other areas i think a tile specific scss is a better approach. Happen to discuss this further.

File: assets/sass/components/_content-tile-small.scss (hunk 1 above)
Comment (jonhindle) [reply assets/sass/components/_content-tile-small.scss:6]: This class `govuk-!-margin-6` should cover it

Comment (BarryGeeMOJ) [reply assets/sass/components/_content-tile-small.scss:6]: turns out the margin style isn't actually needed

File: assets/sass/components/_content-tile-small.scss (hunk 2 above)
Comment (BarryGeeMOJ) [reply assets/sass/components/_content-tile-small.scss]: relocated class into home-content. I think this needs refactoring to break tile styles out into its own file.

**Review by jonhindle — APPROVED** [review]
lgtm

**Pull request merged**